sdk/core/core/src/az_span.c

```diff
@@ -515,7 +521,8 @@ AZ_NODISCARD az_result az_span_append_i64toa(az_span destination, int64_t source
 
   if (source < 0)
   {
-    AZ_RETURN_IF_FAILED(az_span_append(destination, AZ_SPAN_FROM_STR("-"), out_span));
+    AZ_RETURN_IF_SPAN_CAPACITY_TOO_SMALL(destination, 1);
```

Comment:
I think the name of this MACRO is kind of subjective....  what does `too small` means?   I think it's not `too clear` :P
What about `AZ_RETURN_IF_NOT_ENOUGH_CAPACITY`

Reply:
Sure, I can change it to that if you think that conveys the intent more clearly :)